Add Devant concepts docs (#21)
wso2/docs-devant: en/docs/administer/inviting-members.md

## Inviting users

> An organization administrator can invite users to the organization by assigning them specific groups. Invited users receive an invitation via email. An invited user must accept the invitation to join the organization and access the resources of that organization.

Reviewer: ```suggestion
An organization administrator can invite users to the organization by assigning them specific groups. Invited users receive an invitation via email. An invited user must accept the invitation to join the organization and access its resources.
```
